src/Libraries/LibAllowList.sol

```diff
@@ -16,6 +16,7 @@ library LibAllowList {
         mapping(address => bool) allowlist;
```

Comment:
could change to `contractAllowList` for consistency

Reply:
updated